src/javascript/app/pages/trade/barriers.js

```diff
@@ -21,7 +21,7 @@ const Barriers = (() => {
         const barriers  = Contract.barriers()[Defaults.get('underlying')];
         const form_name = Contract.form();
 
-        if (barriers && form_name && Defaults.get('formname') !== 'risefall') {
+        if (barriers && form_name && !/risefall|reset/i.test(Defaults.get('formname'))) {
```

Comment:
probably we can ask API not to return barrier in contracts_for for reset if it's not needed for calling proposal.

Reply:
can you add a TODO comment to remove reset from here when API stops sending it so we don't forget?